common/build/build-common/api-extractor-base.json

```diff
@@ -0,0 +1,218 @@
+{
+	"$schema": "https://developer.microsoft.com/json-schemas/api-extractor/v7/api-extractor.schema.json",
+
+	"compiler": {
+		"tsconfigFilePath": "<projectFolder>/tsconfig.json"
+	},
+
+	/**
+	 * Specifies the .d.ts file to be used as the starting point for analysis. API Extractor analyzes the symbols exported by this module.
+	 * Can be overridden as necessary for packages with different build structures.
+	 *
+	 * See: <https://api-extractor.com/pages/configs/api-extractor_json/#mainentrypointfilepath>
+	 */
+	"mainEntryPointFilePath": "<projectFolder>/dist/index.d.ts",
+
+	/**
+	 * Configures how the API report file (*.api.md) will be generated.
+	 */
+	"apiReport": {
+		"enabled": true,
+		"reportFileName": "<unscopedPackageName>.api.md",
+		"reportFolder": "<projectFolder>/api-report/",
+		"reportTempFolder": "<projectFolder>/_api-extractor-temp/"
+	},
+
+	/**
+	 * Configures how the doc model file (*.api.json) will be generated.
+	 */
+	"docModel": {
+		"enabled": true,
+		"apiJsonFilePath": "<projectFolder>/_api-extractor-temp/doc-models/<unscopedPackageName>.api.json"
+	},
+
+	/**
+	 * Configures how the .d.ts rollup file will be generated.
+	 */
+	"dtsRollup": {
+		"enabled": true,
+		"alphaTrimmedFilePath": "<projectFolder>/dist/<unscopedPackageName>-alpha.d.ts",
```

Comment:
@sonalideshpandemsft it just occurred to me that I should check what is the most convenient naming schema from the tooling perspective, as you're working on that side of things. We're not locked into this `<package-name>-<scope>` scheme.

Would it be better to omit the package name from these?